libpod/oci_attach_linux.go

```diff
@@ -259,7 +260,7 @@ func redirectResponseToOutputStreams(outputStream, errorStream io.Writer, writeO
 				}
 			}
 		}
-		if er == io.EOF {
+		if er == io.EOF || errors.Is(er, syscall.ECONNRESET) {
```

Comment:
Could you also do a `errors.Is(er, io.EOF)`. It probably works as is but who knows if it will continue in the future.

Reply:
According to the godoc this is not necessary:

> EOF is the error returned by Read when no more input is available. (Read must return EOF itself, not an error wrapping EOF, because callers will test for EOF using ==.)

However I agree that using errors.Is() is a good idea and definitely doesn't hurt here.

Reply:
fixed